job/scriptsGen/normalizeSteps.js

```diff
@@ -190,22 +190,18 @@ function __generateRuntimeInfo(task, buildJobId, buildScriptsDir,
         task.runtime.options.imageName, task.runtime.options.imageTag),
       shouldPullTaskContainerImage: task.runtime.options.pull,
       taskContainerCommand: util.format('%s %s %s',
-        global.config.taskContainerCommand,
+        path.join(buildScriptsDir, task.buildScriptFileName),
         path.join(buildScriptsDir, task.taskScriptFileName),
         path.join(buildStatusDir, 'job.env')
       ),
       taskContainerName: containerName
     };
 
-    // Make this override the default when the below OS support this.
-    if (!_.contains(['WindowsServer_2016', 'macOS_10.12'],
-      global.config.shippableNodeOperatingSystem)) {
-      taskContainerEnvs.taskContainerCommand = util.format('%s %s %s',
-        path.join(buildScriptsDir, task.buildScriptFileName),
-        path.join(buildScriptsDir, task.taskScriptFileName),
-        path.join(buildStatusDir, 'job.env')
-      );
-    }
+    // Windows expects powershell scripts to be called with
+    // powershell.exe explicitly
+    if (global.config.shippableNodeOperatingSystem === 'WindowsServer_2016')
```

Comment:
We should maybe just add this in node? https://github.com/Shippable/node/blob/master/initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1#L40

Reply:
Discussed with @scriptnull. This will not result in the behavior we want and we are already hard-coding powershell as a shell for reqExec. So this is okay.